styles/custom-bootstrap/_spacing.scss

```diff
@@ -0,0 +1,62 @@
+$spacer: 4px;
+
```

Comment:
SuperMinor: Since we are redefining everything anyway, can we write our own styles or connect another library?

Reply:
We can use another style library. But It needs to discuss.